helpers/operator.go

```diff
@@ -88,32 +88,41 @@ func BootstrapTykEnterprisePortalSecret() error {
 	}
 
 	for _, value := range secrets.Items {
-		if data.AppConfig.EnterprisePortalSecretName == value.Name {
+		if data.AppConfig.EnterprisePortalSecretName == value.Name ||
+			data.AppConfig.DeveloperPortalSecretName == value.Name {
 			err = clientset.CoreV1().Secrets(data.AppConfig.TykPodNamespace).
 				Delete(context.TODO(), value.Name, v1.DeleteOptions{})
 			if err != nil {
 				return err
 			}
-			fmt.Println("A previously created enterprise portal secret was identified and deleted")
+			fmt.Println("A previously created portal secret was identified and deleted")
 			break
 		}
 	}
 
-	err = CreateTykEnterprisePortalSecret(clientset)
-	if err != nil {
-		return err
+	if data.AppConfig.EnterprisePortalSecretName != "" {
```

Comment:
If both `EnterprisePortalSecretName` and `DeveloperPortalSecretName` are populated with some values, we'll create both secrets. I think we should only go with the new name: `DeveloperPortalSecretName`

Reply:
Given that this is controlled at chart level, shouldn't we leave the option to use both?

Reply:
Once we merge this code piece to the main branch, only new helm-charts, which only provides the new `DeveloperPortalSecretName` option, will use this code piece. so, in my opinion, no need to maintain two versions of the environment variables.

Reply:
Makes sense, implemented.